src/locales-check.js

```diff
@@ -0,0 +1,16 @@
+const getLocalesFiles = require('./get-locales-files');
+const getLocalesStructure = require('./get-locales-structure');
+const getLocalesMissingKeys = require('./get-locales-missing-keys');
+const logLocalesMissingKeys = require('./log-locales-missing-keys');
+
+function localesCheck(path, ci = false) {
+  const localesFiles = getLocalesFiles(path);
+  const localesStructure = getLocalesStructure(Object.values(localesFiles));
+  const missingKeys = getLocalesMissingKeys(localesStructure, localesFiles);
+  if (Object.keys(missingKeys).length > 0) {
+    logLocalesMissingKeys(missingKeys);
+    if(ci) { process.exit(1) };
+  }
+}
```

Comment:
futurology: if we ever want to experiment with using this for an eslint rule, it might be useful to export something like this function but only to compare one locale with the merged locale structure

Reply:
good idea!